src/Divider.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
+import { space, width, borderColor } from 'styled-system'
+import PropTypes from 'prop-types'
+
+const Divider = styled.hr`
+  ${space}
+  ${width}
+  border: 0;
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  ${borderColor}
+`
+
+Divider.displayName = 'Divider'
+
+const numberStringOrArray = PropTypes.oneOfType([
+  PropTypes.number,
+  PropTypes.string,
+  PropTypes.array
+])
+
+Divider.propTypes = {
```

Comment:
I recently added a `propTypes` export for styled-system that you could use to cut down the repeated code here – https://github.com/jxnblk/styled-system#proptypes